sql/6_1_0-to-7_0_0_upgrade.sql

```diff
@@ -922,3 +922,11 @@ UPDATE `list_options` SET `activity` = '1' WHERE `list_id` = 'race' AND `title`
 UPDATE `list_options` SET `activity` = '1' WHERE `list_id` = 'race' AND `title` = 'Korean';
 UPDATE `list_options` SET `activity` = '1' WHERE `list_id` = 'race' AND `title` = 'Asian Indian';
 #Endif
+
+#IfNotIndex patient_history idx_pid
```

Comment:
fix (looks like it got inverted :) ):
```
#IfNotIndex patient_history pid_idx
```

Reply:
You know I just stared at this over and over again and couldn't figure out what you were saying, had to copy and paste the line and then my eyes saw the difference, lol.